.husky/pre-commit

```diff
@@ -0,0 +1,3 @@
+npm run fix:code
+npm run fix:prettier
```

Comment:
Isn't prettier ran through eslint when running the plugins?

Reply:
I run those scripts to test it., and some times eslint make changes after ordering the import. Also CSS and JSON are not included into the `fix:code` command:
```
    "check:code": "eslint ./src --ext .ts,.tsx,.js,.jsx",
    "fix:code": "npm run check:code -- --fix",
```